app/controllers/admin/organizations_controller.rb

```diff
@@ -96,6 +96,13 @@ def update_active
       redirect_to :back, notice: "Updated #{@organization.name}"
     end
 
+    def update_waiting_list
+      @organization.update_attribute(:on_waiting_list, params[:on_waiting_list])
+
+      NotifyOrganizationOffWaitingList.perform(organization: @organization)
```

Comment:
Conventionally this would be if then else style:
```
if NotifyOrganizationOffWaitingList.perform(organization: @organization)
   redirect_to :back, notice: "Updated #{@organization.name}"
else 
   redirect_to :back, alert: "Error message ..."
end
```